### ISSUE TITLE
Abnormal response to growth hormone stimuation test

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6483]
**Preferred term label:**
This is a synonym of the existing term
https://hpo.jax.org/app/browse/term/HP:0000824

**Synonyms**

**Definition (free text, please give PubMed ID)**
Insufficient responses to growth hormone (GH) provocation tests. GH deficiency is defined as a serum peak GH concentration <10 ng/mL on provocation with a combination of at least two separate stimulation tests.
PMID:25914878

Comment
GH stimulation tests are performed with various stimuli, such as insulin, L-dopa, arginine, and growth hormone releasing hormone (GHRH).10 The test using insulin as a stimulus is classically recommended, but this test has potential risks such as hypoglycemia. 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done
